blocks/columns/columns.css

```diff
@@ -578,6 +578,11 @@ main .columns > div > div > h4:first-child {
     gap: 120px;
   }
 
+  .content-band.columns > div > div {
```

Comment:
Can you add `.icon-list` here too?

Reply:
Sounds good, will do.

Reply:
Can you change this to `.columns.content-band.icon-list > div > div` in order to match with existing `.columns.content-band.icon-list > div` selector that controls the column gap.